runtime/darwinia/src/pallets/bridge_dispatch.rs

```diff
@@ -0,0 +1,32 @@
+pub use pallet_bridge_dispatch::Instance1 as S2sBridgeDispatch;
+
+// --- paritytech ---
+use bp_messages::{LaneId, MessageNonce};
+use frame_support::traits::Contains;
+use pallet_bridge_dispatch::Config;
+// --- darwinia-network ---
+use crate::*;
+use crab_messages::FromCrabEncodedCall;
+
+pub struct S2sCallFilter;
+impl Contains<Call> for S2sCallFilter {
+	fn contains(_c: &Call) -> bool {
+		// matches!(
+		// 	c,
+		// 	Call::Substrate2SubstrateBacking(to_substrate_backing::Call::unlock_from_remote(..))
+		// )
+		false
```

Comment:
Include `system.Remark` for now for testing.

Reply:
Is this issue solved?

Reply:
Overwritten by force push.